core/internal/cltest/README.md

```diff
@@ -0,0 +1,4 @@
+// Importing cltest should only be for top level black box (X_test package) integration tests, 
+// for example testing against the Application object itself. 
+// Consider using a smaller scoped package: utils or testutils/* if possible. 
+package cltest
```

Comment:
Did you intend to make this a `doc.go` file instead? Like the idea either way though.

Reply:
yeah, fixed